src/sagemaker/amazon/common.py

```diff
@@ -89,6 +108,46 @@ def write_numpy_to_dense_tensor(file, array, labels=None):
         _write_recordio(file, record.SerializeToString())
 
 
+def write_numpy_to_sparse_tensor(file, array, labels=None):
```

Comment:
Should this be write_spmatrix_to_sparse_tensor ?

Reply:
+1

Reply:
Ok, fixed.